articles/openshift/oshift-how-add-domains-proxy-whitelist.md

```diff
@@ -17,7 +17,7 @@ toc_mdlink: oshift-how-add-domains-proxy-whitelist.md
 
 ## Overview
 
-This article outlines the necessary steps to whitelist domain names within the Squid proxy. Squid is an enabled service within v3.11 OpenShift clusters deployed with connectivity to government community networks (such as: HSCN, PSN, Janet). Whitelisted domains can be accessed on the internet via the proxy, enabling you to request external resources/data on nodes that previously only had connectivity to government community networks.
+This article outlines the necessary steps to whitelist domain names within the Squid proxy. Squid is an enabled service within v3.11 OpenShift clusters deployed with connectivity to government community networks (such as: HSCN, PSN, Janet). Whitelisted domains can be accessed on the internet via the proxy, enabling you to request external resources/data on nodes that previously only had connectivity to government community networks. Added domains should be scrutinised as the relevant authority of the community network may require you to submit documentation regarding these for you to receive accreditation.
```

Comment:
I'm happy with the general gist of this. @suehighmoor or @stevedicko may want to review it too though.